bin/plot_krun_results

```diff
@@ -677,7 +677,7 @@ def draw_page(is_interactive, executions, cycles_executions,
             if not inset_collides:
                 # Create and style the inset.
                 inset = add_inset_to_axis(fig, axis, rect)
-                inset.set_ylim([y_min, y_max])  # Same scale as larger subplot.
+                inset.set_ylim([y_min, y_max])
```

Comment:
The undo hasn't worked, as it's killed the comment. Let's put the comment back in so that the PR has a tight focus.